src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -74,12 +74,12 @@ public function load(array $configs, ContainerBuilder $container)
 
     public function prepend(ContainerBuilder $container)
     {
-        if (!$container->hasParameter('kunstmaan_media.upload_dir')) {
+        if (!$container->hasParameter('kunstmaan_media.upload_dir')) {  // Not used, deprecate?
             $container->setParameter('kunstmaan_media.upload_dir', '/uploads/media/');
         }
 
         $twigConfig = [];
-        $twigConfig['globals']['upload_dir'] = $container->getParameter('kunstmaan_media.upload_dir');
+        $twigConfig['globals']['upload_dir'] = $container->getParameter('kunstmaan_media.upload_dir'); // Not used, deprecate?
```

Comment:
The parameter might not be used, but the twig global still is? So do we need to replace that value somehow or is that also a variable that needs to be deprecated?

Reply:
I cannot find a reference to the twig global, but i have removed the comment. It is not necessary to remove this.